js/classes/entities/midground/drops/food_drop.js

```diff
@@ -53,6 +53,13 @@ class FoodDrop {
         ASSET_MGR.playSFX(sfx.path, sfx.volume);
 
         switch (this.type) {
+            case FoodItem.ROAST_TURKEY:
```

Comment:
Ah I see so max health only increases with roast turkey. Very good! You probably will want to make it hard to get multiple times since you can literally farm any zone right now for food. You may have already considered this. Are you making the big snake drop it?